public/app/plugins/panel/stat/StatPanel.tsx

```diff
@@ -69,10 +69,11 @@ export class StatPanel extends PureComponent<PanelProps<StatPanelOptions>> {
   };
 
   getValues = (): FieldDisplay[] => {
-    const { data, options, replaceVariables } = this.props;
+    const { data, options, replaceVariables, fieldConfig } = this.props;
 
     return getFieldDisplayValues({
-      ...options,
+      fieldConfig,
+      fieldOptions: options.fieldOptions,
```

Comment:
Not super wild about this, I'm considering moving the rest of fieldOptions into PanelModel as well, but don't have any strong argument for or against. Feels like this si standardised property, so probably could live in the PanelModel

Reply:
Currently standard field display options are rendered "together" with other field options that are custom for a given panel. Probably the standard ones (FieldDisplayEditor) could be rendered in a collapsible section as other standard config options we have, and only the custom options would be rendered below. Not sure - any thoughts around this @torkelo @ryantxu ?

Reply:
I don't think the other parts should be on the PanelModel.  I think they can continue to exist as a "common" property used by many panels.  The complexity and logic is kinda limited.

I'm not sure the best name -- the remaining values are about how to select the fields for display